fuseoverlayfs.go

```diff
@@ -494,10 +497,16 @@ func (o *snapshotter) mounts(s storage.Snapshot) []mount.Mount {
 	}
 
 	options = append(options, fmt.Sprintf("lowerdir=%s", strings.Join(parentPaths, ":")))
+	if mapping, ok := info.Labels["containerd.io/snapshot/uidmapping"]; ok {
+		options = append(options, fmt.Sprintf("uidmapping=%s", mapping))
+	}
+	if mapping, ok := info.Labels["containerd.io/snapshot/gidmapping"]; ok {
+		options = append(options, fmt.Sprintf("gidmapping=%s", mapping))
+	}
 	options = append(options, commonMountOptions...)
 	return []mount.Mount{
 		{
-			Type:    "fuse3." + fuseoverlayfsBinary,
+			Type:    "fuse." + fuseoverlayfsBinary,
```

Comment:
Why?

Reply:
I took it out because my OS did not support fuse3 for some reason. I'll add it back in!